website/docs/enterprise/getting-started/join-cluster-azure-flux.mdx

```diff
@@ -57,14 +57,13 @@ If you already have Flux running, you can create the service account in your fle
 <details><summary>Expand to see role manifests</summary>
 
 ```yaml
+---
```

Comment:
```suggestion
```

Reply:
The multi document YAML separator `---` is usually omitted before the first document (and after the last one).